src/main/java/world/bentobox/bentobox/util/Util.java

```diff
@@ -578,7 +577,7 @@ public static String translateColorCodes(@NonNull String textToColor) {
      */
     @NonNull
     public static String stripSpaceAfterColorCodes(@NonNull String textToStrip) {
-        Validate.notNull(textToStrip, "Cannot strip null text");
+        if (textToStrip == null) return "";
```

Comment:
Avoid the apache commons function as it may not be included in the server code now.